app/components/mandatenbeheer/mandataris/mandataris-edit-prompt.hbs

```diff
@@ -1,44 +1,48 @@
-<div class="au-o-grid au-o-grid--small">
-  <div class="au-o-grid__item au-u-1-2">
-    <AuContent @skin="small">
-      <AuHeading @level="4" @skin="5">Wijziging huidige situatie</AuHeading>
-      <p>Indien de mandataris wordt vervangen, of de huidige
-        <strong>status, fractie, rangorde of bevoegdheid</strong>
-        verandert:
-      </p>
-      <ul>
-        <li>beëindig dit mandaat door de einddatum in te vullen;</li>
-        <li><strong>start een nieuw mandaat</strong>
-          met de gewijzigde situatie.</li>
-      </ul>
-      <AuButton {{on "click" this.terminate}}>Wijzig einde mandaat</AuButton>
-    </AuContent>
+{{#if this.notEditing}}
```

Comment:
can be `{{#if (not this.editMode)}}` so you don't need the extra getter that's just a negation